lib/diagnostic.rb

```diff
@@ -13,28 +13,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    (enum).inject { |sum, n| sum + n }
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
-  def some_odd(numbers)
-    # Your code goes here
+  def some_odd(numbers)   # one or more
+    numbers.any? { |num| num % 2 != 0 }
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    numbers.all? { |num| num % 2 == 0}
```

Comment:
Correct, but as above, there's a method you could have used as a shortcut, `.even?`
e.g.
`numbers.all?(&:even?)`